app/src/components/IncomeRow.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactElement } from 'react'
```

Comment:
Notes for reviewers: This component is broken out from `/income` primarily so that we can call it as a [dynamic import](https://nextjs.org/docs/advanced-features/dynamic-import) and avoid hydration issues.